kpm/lib/kpm/tasks.rb

```diff
@@ -598,6 +598,43 @@ def tenant_config
         end
 
 
+        method_option :export,
+                      :type    => :string,
+                      :default => nil,
+                      :desc    => 'export account for a provided id.'
+        method_option :config_file,
+                      :type    => :string,
+                      :default => nil,
+                      :desc    => 'Yml that contains killbill api connection and DB connection'
+        method_option :log_dir,
+                      :type    => :string,
+                      :default => nil,
+                      :desc    => '(Optional) Log directory if the default tomcat location has changed'
+        desc 'diagnostic', 'exports and \'zips\' the account data, system, logs and tenant configurations'
+        def diagnostic
+          logger.info 'Please wait processing the request!!!'
+          begin
```

Comment:
It looks like the indentation is off, making it hard to follow where the block ends.